scitran/reaper/dicom_file_reaper.py

```diff
@@ -100,7 +100,7 @@ def main():
         (('path',), dict(help='path to DICOM files')),
     ]
     optional_args = [
-        arg_parser.add_argument('-d', '--destructive', action='store_true', help='delete data after reaping')
+        (('-d', '--destructive'), dict(action='store_true', help='delete data after reaping'))
```

Comment:
Not sure if it was always broken, but sure didn't work for me on a clean install from master because arg_parser is not included here.